### PR TITLE
Make android app compile on jenkins build server

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: Integreat/Integreat.Droid/Resources/values/styles.xml [hunk 1]
```diff
@@ -24,6 +24,5 @@
     </style>
     
     <style name="SplashTheme.Base" parent="Theme.AppCompat.Light.NoActionBar">
-        <item name="android:windowBackground">@drawable/splash_screen</item>
```
Comment (maxammann) [inline Integreat/Integreat.Droid/Resources/values/styles.xml]: Splash screen is not compatible for some reason: https://github.com/bamlab/generator-rn-toolbox/issues/144

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+pipeline {
+    agent any
+
+    stages {
+        stage('Build') {
+            steps {
+                sh 'mkdir packages && cd packages && nuget install ../Integreat/Integreat.Droid/packages.config'
+                sh 'msbuild Integreat.sln /p:AndroidSdkDirectory="/opt/android-sdk/" || true'
```
Comment (maxammann) [inline Jenkinsfile]: First execution fails with SIGSEV

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+                sh 'msbuild Integreat/Integreat.Droid/Integreat.Droid.csproj /p:AndroidSdkDirectory="/opt/android-sdk/" /target:SignAndroidPackage'
```
Comment (maxammann) [inline Jenkinsfile]: SignAndroidPackage target is not available in Localisation project so we only build the Droid project.

File: Integreat/Integreat.Droid/Resources/values/styles.xml (hunk 1 above)
Comment (thimmy687) [reply Integreat/Integreat.Droid/Resources/values/styles.xml]: maybe we can add this directly in code 🤔

**Comment by maxammann** [comment]
Yes or wie decide agains them: https://androiduipatterns.com/splash-screen-with-any-other-name-is-still-a-splash-screen-and-they-suck-bab7f3b5850c

**Comment by thimmy687** [comment]
@maxammann can we add the Jenkins here next to appveyor?

So we can see what is wrong and maybe fix it

**Comment by maxammann** [comment]
@thimmy687 its already there: https://build.integreat-app.de/job/app-cross_platform/
It compiles branches with a Jenkinsfile

You can also login with our integreat account

**Comment by maxammann** [comment]
This can be abandoned. I think I already removed mono from the build server.

**Pull request closed without merging**